loris/resolver.py

```diff
@@ -61,13 +65,40 @@ def resolve(self, ident):
             ident (str):
                 The identifier for the image.
         Returns:
-            (str, str): (fp, format)
+            ImageInfo: Partially constructed ImageInfo object 
```

Comment:
Okay, I'm at a loss for why the final two tests aren't working. I believe it's something to do with the testing framework, rather than the code.  Have added you (@bcail) to my repo if you could take a look at the errors?